frontend/src/main/scala/bloop/data/WorkspaceSettings.scala

```diff
@@ -38,7 +36,8 @@ import com.github.plokhotnyuk.jsoniter_scala.macros.{JsonCodecMaker, CodecMakerC
 case class WorkspaceSettings(
     semanticDBVersion: Option[String],
     supportedScalaVersions: Option[List[String]],
-    refreshProjectsCommand: Option[List[String]]
+    refreshProjectsCommand: Option[List[String]],
+    traceProperties: TraceProperties
```

Comment:
This field needs to be optional so that it doesn't have to appear in `bloop.settings.json`